app/Http/Controllers/V1/ReservationController.php

```diff
@@ -88,6 +88,7 @@ public function store(ReservationRequest $request)
     public function update(ReservationRequest $request, Reservation $reservation)
     {
         abort_if($reservation->is_not_yet_approved, 500, __('validation.asset_modified'));
+        abort_if($reservation->start_time->subMinutes(30) > Carbon::now('+07:00'), 500, __('validation.asset_modified_time'));
```

Comment:
Di comment di atas kan sudah clear ya, use case-nya seperti apa.

Kalau begitu, logic di line 91 seharusnya seperti ini bukan?
```suggestion
        abort_if(Carbon::now('+07:00') > $reservation->start_time->subMinutes(30), 500, __('validation.asset_modified_time'));
```

Reply:
Betul kang ke balik operator nya harus abort if mencari nilai false 👍